iree/turbine/kernel/lang/__init__.py

```diff
@@ -13,6 +13,7 @@
 )
 
 from .._support.dtype import (
+    bf16,
```

Comment:
nit: Can you also add `DataType` class to this list, so we won't need to import `._support.dtype` which is supposed to be internal module.

Reply:
sure